Library/Formula/aws-sdk-cpp.rb

```diff
@@ -0,0 +1,70 @@
+class AwsSdkCpp < Formula
+  desc "AWS SDK for C++"
+  homepage "https://github.com/aws/aws-sdk-cpp"
+  url "https://github.com/aws/aws-sdk-cpp/archive/0.9.6.tar.gz"
+  sha256 "cf2dad62e50c5c5cff88a4ad959af3aa402aa7e8aa1692bdf807052c0fe4aeb9"
+  head "https://github.com/aws/aws-sdk-cpp.git"
+
+  ALL_CLIENTS = [
```

Comment:
The build from source for all clients takes 20-30 minutes on my 2015 mbp, which is why I originally allowed for any set of clients. Maybe a better alternative is something like --minimal --standard and --most. Maybe something like:

--minimal: s3, config, iam, ec2
--standard: --minimal + rds, redshift, lambda, cloudfront, cloudtrail, glacier
--most: --standard + route53, elasticbeanstalk, elasticloadbalancing, ??